subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1784,8 +1792,9 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_twt_ops,
 		"<setup_cmd: 0: Request, 1: Suggest, 2: Demand>\n"
 		"<dialog_token: 1-255> <flow_id: 0-7> <responder: 0/1> <trigger: 0/1> <implicit:0/1> "
 		"<announce: 0/1> <twt_wake_interval: 1-262144us> <twt_interval: 1us-2^31us>.\n",
```

Comment:
I believe the comma at the end of the line should be removed since an extra line was added.

Reply:
Updated